js/modules/k6/http/http_test.go

```diff
@@ -283,6 +279,61 @@ func TestRequest(t *testing.T) {
 			assert.EqualError(t, err, "GoError: invalid character '<' looking for beginning of value")
 		})
 	})
+	t.Run("TLS", func(t *testing.T) {
+		t.Run("cert_expired", func(t *testing.T) {
+			_, err := common.RunString(rt, `http.get("https://expired.badssl.com/");`)
+			assert.EqualError(t, err, "GoError: Get https://expired.badssl.com/: x509: certificate has expired or is not yet valid")
+		})
+		t.Run("tls10", func(t *testing.T) {
```

Comment:
These versions are nigh identical and would do better templated than copypasted.

Reply:
Ok, will look into that.